views/tasks/active-officers-details.html

```diff
@@ -65,8 +71,8 @@ <h1 class="govuk-heading-l">Check the officers' details</h1>
       {{ govukRadios({
         classes: "govuk-radios",
         idPrefix: "officers",
-        name: "officers",
-        errorMessage: directorDetailsError,
+        name: "activeOfficers",
```

Comment:
This is fine but also delivers what I was working on (not very much!) - https://companieshouse.atlassian.net/browse/BI-10191. So we can mark that task as done too.

Reply:
Should still need the data event ids added, I can put that in this as well if that's easier?